### ISSUE TITLE
how to use PINO for inverse problem of pde 

### ISSUE THREAD

**Issue by xgxgnpu** [shawnrosofsky/PINO_Applications#1]

**Comment by xgxgnpu** [comment]
PINO is a very nice framework to learn pde operator, that is forward problem for pde, but how to use PINO for inverse problem of pde,such as parapmeter identification of pde

**Comment by shawnrosofsky** [comment]
While I have not tried using PINOs for inverse problems, it should be a fairly straight forward to do so.  I believe all you need to do is register additional parameters into the network to act as the PDE coefficients and use them in the equation defined physics loss function (I typically called FDM_(equation name) in the Jupyter notebooks).  

This should be the same as in the case as a traditional PINN.  For an example of how to do this in PyTorch, [see this Jupyter notebook](https://github.com/jayroxis/PINNs/blob/master/Burgers%20Equation/Burgers%20Identification%20(PyTorch).ipynb).
